packages/block-library/src/embed/editor.scss

```diff
@@ -1,3 +1,11 @@
+// Apply max-width to floated items that have no intrinsic width
+.block-editor-block-list__block[data-type^="core-embed"][data-align="left"] .block-editor-block-list__block-edit,
+.block-editor-block-list__block[data-type^="core-embed"][data-align="right"] .block-editor-block-list__block-edit {
```

Comment:
I'm not completely clear on this though. Maybe `core/embed` is correct. I just saw where thats used for the generic embed? Everything else appears to use something like `core-embed/youtube`. 

Are the generic embeds the only thing that doesn't have an intrinsic width?